2023/91-sae-opensource.md

```diff
@@ -0,0 +1,43 @@
+# Sae Ra Germaine:  Exploring the Open Source Landscape in Australia
+
+## Upcoming Events
+Join our Meetup group for more events!
+https://www.meetup.com/data-umbrella
+
+## Key Links
+- Transcript: https://github.com/data-umbrella/event-transcripts/blob/main/2023/91-sae-opensource.md
+- Meetup Event: https://www.meetup.com/data-umbrella/events/296383380/
+- Video: 
+- GitHub repo:  
+- Transcriber:  ? [needs a transcriber]
+
+## Resources
+None
+
+## About the Event
+Australia is geographically located far away from Europe and North America where the majority of open source maintainers in the Python and scientific python spaces are located. 
+How does that impact Australia's participation in open source? We will explore this......Our little corner of the world - Open Source landscape and the community that supports it in Australia, with a smidge of GLAM[our].
+
+```
+## Timestamps
+00:00 Help us add timestamps
+```
+https://github.com/data-umbrella/event-transcripts/issues/92
+
+## About the Speaker
+Sae Ra Germaine is the Deputy CEO and Manager, Member & Academic Services, VALA President, and a Linux Australia Council Member. 
+Sae Ra is a strong advocate for IT and Open Source especially in the Library industry. As a side gig in her not so day job Sae Ra is a serial conference organiser running the inaugural Everything Open Conference and 5 linux.conf.au’sIn the library space Sae Ra has been key in bringing the first installation of the open source Library Management Platform FOLIO to Australia.
+
+- X: https://twitter.com/ms_mary_mac
+- LinkedIn: https://www.linkedin.com/in/srgermaine/
+
+#opensource
+
+## Video
+<a href="http://www.youtube.com/watch?feature=player_embedded&" target="_blank"><img src="http://img.youtube.com/vi/e9fB8NQV6Nc/0.jpg"
+alt="Exploring the Open Source Landscape in Australia" width="50%" /></a>
+
```

Comment:
<a href="http://www.youtube.com/watch?feature=player_embedded&v=4m1jVQdUTQo" target="_blank"><img src="http://img.youtube.com/vi/4m1jVQdUTQo/0.jpg"
alt="Exploring the Open Source Landscape in Australia" width="50%" /></a>